tests/testSdssShape.py

```diff
@@ -127,6 +189,7 @@ def suite():
     suites = []
     suites += unittest.makeSuite(SdssShapeTestCase)
     suites += unittest.makeSuite(SdssShapeTransformTestCase)
+    suites += unittest.makeSuite(PsfSdssShapeTransformTestCase)
```

Comment:
Done. This was [messier than I expected](https://github.com/lsst/meas_base/pull/45/commits/0ee4d5ca15449ae25740ee339408f6d2280b4573), but I think it's worth merging.